### ISSUE TITLE
Focus state on neutral header variant is not accessible

### ISSUE THREAD

**Issue by daniellecorke** [ONSdigital/design-system#3204]
### Background/Issue
Several issues with the [Neutral header for multi-coloured brand logo](https://service-manual.ons.gov.uk/design-system/components/header/example-header-neutral-for-multicoloured-logo) variant have been identified. 

1. The focus state for the links in the Neutral header is not accessible. The text is supposed to be black and not white.

![Image](https://github.com/ONSdigital/design-system/assets/102512949/48696a7d-8c26-4cf9-9770-44c1e06a6681)

2. Some items have a double line when focused and others have one. The ONS logo cannot be seen when in focus state. We do not display the submenu in the example. 

![Image](https://github.com/ONSdigital/design-system/assets/102512949/44d9fc73-c935-441e-9d02-26389bd7e064)

### Tasks

This ticket covers fixing the bugs highlighted in the ticket. Further discussions are required to discuss a full re-design of the Header component. 

1. Ensure that text in focus state is black. 
2. Ensure that one single line is displayed in focus state. 
3. Ensure that the ONS logo can be displayed in focus state. 

### Acceptance Criteria

If possible, each bug is fixed and tested.

**Comment by SriHV** [comment]
Had a chat with @JNATION85. We are doubtful above the use case of [Neutral header for multi-coloured brand logo](https://service-manual.ons.gov.uk/design-system/components/header/example-header-neutral-for-multicoloured-logo) and hover line when focussed(hover line turns to white instead of black, this however blends with the header with the current styling) which needs further discussion.
